Engine/Src/Ancona/System/iOS/Log.cpp

```diff
@@ -0,0 +1,11 @@
+#include <iostream>
+#include <asl.h>
+
+#include <Ancona/System/Log.hpp>
+
+using namespace ild;
+
+void LogControls::_log(const std::string & msg)
+{
+    asl_log(NULL, NULL, ASL_LEVEL_ERR, ("[Ancona] " + msg).c_str());
```

Comment:
This should be an Objective-C++ file and use NSLog instead since asl_log is deprecated